### PR TITLE
Update readme

### PR DESCRIPTION
**Pull request by FallenChromium**
Depends on: https://github.com/ostis-ai/ostis-project/pull/1 (demo .gif won't work until this is merged)
- Fixes installation problem on Ubuntu 20.04, but additional tests on a clean OS needed - there might be other problems with the `npm` installation procedure.
- Revamps README to make it more attractive and easy to read

### CODE REVIEW

File: README.md
Comment (NikitaZotov) [inline README.md:5]: I'll propose you what you can write this soon :)

File: README.md
Comment (NikitaZotov) [inline README.md:50]: with kb?

File: README.md
Comment (NikitaZotov) [inline README.md:13]:
```suggestion
We are providing `ostis/sc-web` Docker image to ease the integration of this UI with existing OSTIS-systems.
```

**Review by NikitaZotov — DISMISSED** [review]

File: README.md
Comment (AlexandrZagorskiy) [inline README.md:59]: Should we add some information about configuration?

**Review by MksmOrlov — APPROVED** [review]

File: README.md
Comment (FallenChromium) [reply README.md:59]: Sounds like a good idea! We should decide which configuration we'll document though, because we have 2 mechanisms for this:
- the `app.py` flags
- the `sc-web.ini` file. 

In case of the former, I do not like the idea of posting the `--help` section of it, though we could note that it's possible to learn more about the script using this flag. In case of the latter - I know literally nothing about this config file, so some suggestions would be appreciated 😄

File: README.md
Comment (FallenChromium) [reply README.md:50]: That's debatable. 
Recently we had a discussion on whether sc-server should be able to start without a KB at all, so it depends on `sc-machine` project rather than this one. Now it can't start without a KB being built, so `up and running` implies having a working KB.

Either way all the necessary KB fragments are populated during the startup of sc-web, so **technically** it should be possible to start it with an empty KB and everything will work fine. Testing needed 🤔

File: README.md
Comment (Liza909) [inline README.md:13]:
```suggestion
We provide the `ostis/sc-web` Docker image to simplify the integration of this UI with existing OSTIS-systems.
```

File: README.md
Comment (KovalM) [inline README.md:1]: Let's add license as in others projects

**Review by KovalM — CHANGES_REQUESTED** [review]

File: README.md
Comment (VasKho) [reply README.md:59]: I think it'd be good, but it's not high priority so we could hold this change for a while

File: README.md
Comment (FallenChromium) [reply README.md:1]: The license of this project is a topic for discussion. I agree that we should add a license, but we don't have one yet, and it looks like a separate PR anyway 🤔

File: README.md
Comment (FallenChromium) [reply README.md:5]: Leaving as-is for now. I agree that we should update this text when the project will become a bit more mature though!

**Review by KovalM — APPROVED** [review]

**Review by NikitaZotov — APPROVED** [review]

**Review by VasKho — APPROVED** [review]
LGTM :+1:

**Review by AlexandrZagorskiy — APPROVED** [review]

**Pull request merged**
